### PR TITLE
Improve visual elements in index page

### PR DESCRIPTION
**Pull request by mixth**
This PR purpose is to improve index page based on the design prototype.

## Description
- Update styling of button and add them to votelog and representative sections
- Add icons for cabinet, senate, and representative
- Refactor two columns of พรรคร่วมรัฐบาล and พรรคฝ่ายค้าน in `/representatives` to new element called `PartyGroupList`
- Apply `PartyGroupList` to index page

## Screenshot
Screenshot_2019-11-15 Home

### CODE REVIEW

File: src/pages/representatives.js [hunk 1]
```diff
@@ -192,11 +152,17 @@ const RepresentativesPage = ({ data }) => {
...
             <h2 style={{ ...cssEngTitle }}>25th House of Representative</h2>
             <h2 style={{ ...cssEngTitle }}>About</h2>
             <p style={{ ...cssPageP }}>
-            เป็นส่วนหนึ่งของฝ่ายนิติบัญญัติ ที่ได้มาจากการเลือกตั้งทั่วไป พ.ศ. 2562 ประกอบด้วยสมาชิก (ส.ส.) 500 คนตามระบบจัดสรรปันส่วนผสม โดย 350 คนเป็นผู้แทนเขต และอีก 150 คนมาจากระบบบัญชีรายชื่อ ฝ่ายรัฐบาลเป็นรัฐบาลผสมพรรคพลังประชารัฐ ส่วนฝ่ายค้านคือพรรคเพื่อไทยและพรรคร่วมฝ่ายค้านอีก 6 พรรค
+              เป็นส่วนหนึ่งของฝ่ายนิติบัญญัติ ที่ได้มาจากการเลือกตั้งทั่วไป พ.ศ.
+              2562 ประกอบด้วยสมาชิก (ส.ส.) 500 คนตามระบบจัดสรรปันส่วนผสม โดย 350
+              คนเป็นผู้แทนเขต และอีก 150 คนมาจากระบบบัญชีรายชื่อ
+              ฝ่ายรัฐบาลเป็นรัฐบาลผสมพรรคพลังประชารัฐ
+              ส่วนฝ่ายค้านคือพรรคเพื่อไทยและพรรคร่วมฝ่ายค้านอีก 6 พรรค
             </p>
```
Comment (p16i) [inline src/pages/representatives.js:166]: Maybe at some point the future, we should separate this text from the template.  For now, it's ok.

@rapee what do you think?

Comment (rapee) [reply src/pages/representatives.js:166]: Yep, this description should be sourced from data. In the next update.

**Comment by rapee** [comment]
@mixth Thanks for this update even though we don't have an issue for it yet :)

**Comment by mixth** [comment]
@rapee Seems like most of the non-complicated Release 1 issues are taken. So, I was looking around to improve anything that may need in the timeline. :D

I forgot to add that, the senate's icon@2x is missing from the Dropbox folder. This affects the sharpness of the icon in the Index page.

**Comment by rapee** [comment]
@mixth Appreciated that 💯 

Will add the icon later.

**Pull request merged**